ci/requirements/environment.yml

```
@@ -7,23 +7,23 @@ dependencies:
   - boto3
   - bottleneck
   - cartopy
-  - cdms2
+  # - cdms2  # TODO: Waiting on python 3.11 support.
   - cfgrib
   - cftime
   - dask-core
   - distributed
   - flox
   - fsspec!=2021.7.0
-  - h5netcdf
-  - h5py
-  - hdf5
+  # - h5netcdf  # TODO: Waiting on python 3.11 support.
+  # - h5py  # TODO: Waiting on python 3.11 support.
+  # - hdf5  # TODO: Waiting on python 3.11 support.
```

there should be `h5py` packages for python 3.11 on `conda-forge`, so at least for the standard environment I think it should be possible to undo this?
```suggestion
  - h5netcdf
  - h5py
  - hdf5
```